Exercise8/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercise8
+{
+    class Program
+    {
+        static bool IsPrimeNumber(int number)
+        {
+            if (number < 2) return false;
+            for (int i = 2; i <= Math.Sqrt(number); i++)
```

Comment:
For better performance, you should write `for (int i = Math.Sqrt(number); i >=2 ; i--)`

Reply:
Done